pyproject.toml

```diff
@@ -42,6 +42,7 @@ package = false
 
 [tool.uv.sources]
 dc-response-builder = { git = "https://github.com/DemocracyClub/dc_response_builder.git", tag = "1.1.3" }
+uk-election-timetables = { git = "https://github.com/DemocracyClub/uk-election-timetables.git", rev = "e5d20edae3f888e4ce8f1519e5c3903b4fc47aa0" }
```

Comment:
Temporarily pulling in the WIP changes from https://github.com/DemocracyClub/uk-election-timetables/pull/52

TODO: Once we have reviewed and merged that, cut a new release, then update this PR to use the released version